pages/how-to-apply.md

```diff
@@ -2,12 +2,10 @@
 permalink: /how-to-apply/
 title: How to apply
 ---
+If you are interested in joining 18F, use the application link below. This page also contains other information about the application process.
 
-18F and the U.S. Digital Service are distinct groups working toward the same goals. 18F is a digital consultancy within the General Services Administration (GSA). We develop products for other agencies, and those agencies pay for our services. In contrast, the U.S. Digital Service is headquartered inside the Office of Management and Budget. They support and coordinate the work of various digital service teams housed in agencies across the federal government.
+##[Apply to join 18F](/joining-18f/pages/apply.html)|
```

Comment:
A bug, how exciting! When I remove the trailing pipe (`|`) at the end of that line, the entire link vanishes in preview. Not yet certain whether that's a markdown bug or _____.

In the meantime, at least, how about the following:

## Apply to join 18F
[Submit an application to join 18F](/joining-18f/pages/apply.html)